### ISSUE TITLE
Expose the line number of the current line to accessibility APIs

### ISSUE THREAD

**Issue by melisaAkel** [melisaAkel/vscode#176]
Original issue: https://github.com/microsoft/vscode/issues/247942

The current line number is displayed in the status bar and can also be obtained by looking at the column of numbers to the left of the editor. However, there is no way to easily and reliably obtain that via accessibility API (e.g. AT-SPI2). I (the Orca maintainer) would like to include the line number on demand to users via the WhereAmI command. Therefore I need a reliable way to get it.

Note that AT-SPI2 currently lacks API specific to this purpose. Determining what that API should be requires input from the GTK maintainers (see https://gitlab.gnome.org/GNOME/gtk/-/issues/7526). That said, an object attribute should do the trick until that's sorted out.

@isidorn Thoughts?
